### PR TITLE
Provide an Aggregator that can condense multiple lightblue requests into a single bulk request

### PR DESCRIPTION
**Pull request by dcrissman**
- LightblueProducer also needed to support BulkRequest

### CODE REVIEW

File: src/main/java/com/redhat/lightblue/camel/LightblueProducer.java [hunk 1]
```diff
@@ -36,4 +40,25 @@ public void process(Exchange exchange) throws Exception {
         }
     }
 
+    /*
+     * TODO I would prefer this method return a LightblueResponse instead of Object,
```
Comment (paterczm) [inline src/main/java/com/redhat/lightblue/camel/LightblueProducer.java:44]: Did you create an issue for both AbstractLightblueDataRequest and AbstractDataBulkRequest to implement the same base or interface?

File: src/main/java/com/redhat/lightblue/camel/request/DataBulkRequestAggregationStrategy.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+/**
+ * Aggregates a collection of {@link AbstractLightblueDataRequest}s into a single {@link DataBulkRequest}.
+ *
+ * @author dcrissman
+ */
+public class DataBulkRequestAggregationStrategy implements AggregationStrategy {
+
+    @Override
+    public Exchange aggregate(Exchange oldExchange, Exchange newExchange) {
```
Comment (paterczm) [inline src/main/java/com/redhat/lightblue/camel/request/DataBulkRequestAggregationStrategy.java:18]: You take request from newExchange and add it to oldExchange - shouldn't the bulk one be new?

File: src/main/java/com/redhat/lightblue/camel/LightblueProducer.java (hunk 1 above)
Comment (dcrissman) [inline src/main/java/com/redhat/lightblue/camel/LightblueProducer.java:44]: https://github.com/lightblue-platform/lightblue-client/issues/174

File: src/main/java/com/redhat/lightblue/camel/request/DataBulkRequestAggregationStrategy.java (hunk 2 above)
Comment (dcrissman) [inline src/main/java/com/redhat/lightblue/camel/request/DataBulkRequestAggregationStrategy.java:18]: I think 'old' and 'new' are confusing words, but those are the words camel uses. Think of 'old' as the collection being built, and new as the item to be added to the collection.

**Pull request merged**